e2e/testdata/root-sync-helm-chart-cr.yaml

```diff
@@ -18,6 +18,11 @@ metadata:
   name: root-sync
   namespace: config-management-system
 spec:
+  override:
+    resources:
+    - containerName: helm-sync
+      cpuRequest: "200m"
+      memoryRequest: "600Mi"
```

Comment:
I was thinking maybe set the override for all tests in the test setup.
Is this artifact used in all the tests that are failing due to OOMKill?

Reply:
Based on the CI dashboard, fixing this one should get the CI mostly green and avoid the buganizer tickets to be filed.